packages/wdio-cli/src/config.js

```diff
@@ -39,6 +39,11 @@ export const SUPPORTED_RUNNERS = [
     ' lambda - https://www.npmjs.com/package/wdio-lambda-runner'
 ]
 
+export const SUPPORTED_BROWSERS = [
```

Comment:
We should not limit this to two browser. Edge as well as Safari Technology Preview have excellent implementation of latest WebDriver spec. Also why not include mobile?

Reply:
For why not to include mobile, my thought is that mobile requires extra setup through Appium, which may not be obvious to newcomers who see mobile and think "sure, why not test on it if it's that easy".

Reply:
All environments require extra steps, see my response https://github.com/webdriverio/webdriverio/pull/3026#issuecomment-434001469